free-programming-cheatsheets-en.md

```diff
@@ -0,0 +1,108 @@
+### Index
+
+* [Algorithms & Data Structures](#algorithms--data-structures)
+* [Android](#android)
+* [Angular](#angular)
+* [Bootstrap](#bootstrap)
+* [C](#c)
+* [C#](#c-sharp)
+* [C++](#c-1)
+* [Git](#git)
+* [iOS](#ios)
+* [Java](#java)
+* [JavaScript](#javascript)
+  * [Next.js](#nextjs)
+  * [NodeJS](#nodejs)
+  * [React](#react)
+  * [React Native](#react-native)
+  * [Redux](#redux)
+* [PHP](#php)
+* [Python](#python)
+  * [Django](#django)
+  * [Flask](#flask)
+
+
+### algorithms--data-structures
+* [Algorithms and Data Structures Cheatsheet](https://algs4.cs.princeton.edu/cheatsheet/) - Princeton
+* [Data Structures Cheatsheet](https://intellipaat.com/mediaFiles/2019/02/Python-Data-structures-cheat-sheet.pdf) - Intellipaat
+
+
+### android
+* [Android Cheatsheet](https://github.com/anitaa1990/Android-Cheat-sheet) - Anita Murthy
+
+
+### angular
+* [Angular Cheatsheet](https://angular.io/guide/cheatsheet) - Angular Dev team
+
+
+### Bootstrap
+* [Bootstrap Cheatsheet](https://hackerthemes.com/bootstrap-cheatsheet/) - Hacker Themes
+* [Bootstrap Cheatsheet PDF](https://websitesetup.org/bootstrap-cheat-sheet/) - Website Setup

```

Comment:
There is no need to specify PDF in the name, it's accessible via the webpage, and has a PNG view available too.

Something will have to be done about the naming convention though to avoid duplicate titles I assume.